.github/workflows/build.yml

```
@@ -139,7 +139,7 @@ jobs:
         uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
 
       - name: Docker Build
-        uses: hashicorp/actions-docker-build@v1
+        uses: hashicorp/actions-docker-build@v2
```

We should pin the GitHub Action for better security.
```suggestion
        uses: hashicorp/actions-docker-build@11d43ef520c65f58683d048ce9b47d6617893c9a #v2
```